bin/generate-npm-tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# Get highest tag published on Github
+HIGHEST_PUBLISHED_TAG=$(git tag --list 2>/dev/null | sort -V | tail -n1 2>/dev/null | sed 's/v//g')
```

Comment:
I think there's a bit of potential of confusion here between Git tags and npm tags 😅 Could we rename this to `HIGHEST_PUBLISHED_VERSION`?

```suggestion
HIGHEST_PUBLISHED_VERSION=$(git tag --list 2>/dev/null | sort -V | tail -n1 2>/dev/null | sed 's/v//g')
```